sintr_worker/lib/basic_info.dart

```diff
@@ -87,13 +86,31 @@ String logKey, Stream<String> stream) async {
         return result;
       }
     }
-    if (line.startsWith(data_prefix)) {
-      // Extract the info and add a result of the format
-      // <millisSinceEpoch>:Ver:<uuid>:<clientId>:<clientVersion>:<serverVersion>:<sdkVersion>
-      result[basic_info_key] = [line.substring(data_prefix.length)];
+    if (line.startsWith('Data :: ~')) {
+      int index = line.indexOf(':Ver:');
+      if (index != -1) {
+        int milliSinceEpoch = int.parse(line.substring(9, index));
```

Comment:
This means that in that on the 20th of November, 2286, in the afternoon, the code will stop working.

:(

[/joke]